### ISSUE TITLE
Running Maxquant on pulsar node

### ISSUE THREAD

**Issue by mthang** [galaxyproteomics/tools-galaxyp#696]
Hi there,
This is just an PR to Galaxy-P maxquant. Currently, Galaxy Australia has cloned this Maxquant and made some modification to include DotNet and updated the maxquant from 1.6.17 to 2.0.3.0 in our maxquant.xml https://github.com/usegalaxy-au/tools-au/blob/master/tools/maxquant/maxquant.xml , so the maxquant can run on the remote pulsar node. We also increase the size of vm.max_map_count on the pulsar to prevent the tool from crashing. 

https://github.com/galaxyproteomics/tools-galaxyp/blob/423304b26e63d23cd8e5fb4c2fb729c5beea1254/tools/maxquant/maxquant.xml#L26

**Comment by bgruening** [comment]
Thanks, @mthang for the update. Are you planning to contribute your modifications back to this repo?

**Comment by mthang** [comment]
@bgruening 
Yes. GA had this discussion and plan, so we are happy to contribute our modifications back to the repo. Can you please let me know what I need to do ?

Best,
Mike

**Comment by bgruening** [comment]
@mthang you push your changes to this repo. The tool wrapper is hosted here and everyone can contribute fixes.

**Comment by bgruening** [comment]
@mthang please see https://github.com/galaxyproteomics/tools-galaxyp/pull/698

Now both versions are diverging I guess.
